src/CatalogService/src/Application/Catalog/Commands/UpdateCatalog/UpdateCatalogCommand.cs

```diff
@@ -2,11 +2,12 @@
 using Catalog.Abstractions;
 using CatalogService.Application.Common.Exceptions;
 using CatalogService.Application.Common.Interfaces;
+using CatalogService.Application.Common.Models;
 using MediatR;
 
 namespace Microsoft.Extensions.DependencyInjection.Catalog.Commands.UpdateCatalog;
 
-public class UpdateCatalogCommand : IRequest<int>
+public class UpdateCatalogCommand : IRequest<CatalogDto>
```

Comment:
Returning the model as a result of the command violates CQS. It might be ok in some rare cases, but it's hard to support in future. Please, send another request to the mediator to get a needed model

Reply:
fixed